loadbalancer/target.go

```diff
@@ -25,13 +25,20 @@ type lbTarget struct {
 func (target *lbTarget) HandleConn(conn net.Conn) {
 	upstreamBackend, err := target.list.Pick()
 	if err != nil {
-		target.logger.Printf("no upstreams available, closing connection from %s", conn.RemoteAddr())
+		target.logger.Info(
+			"no upstreams available, closing connection",
+			zap.String("remote_addr", conn.RemoteAddr().String()),
+		)
 		conn.Close() //nolint: errcheck
 
 		return
 	}
 
-	target.logger.Printf("proxying connection %s -> %s", conn.RemoteAddr(), upstreamBackend.address)
+	target.logger.Info(
```

Comment:
I will probably change this and on line 61 to be `.Debug`, otherwise it's super-noisy

Reply:
Done!